control-plane/pkg/apis/messaging/v1beta1/kafka_channel_validation.go

```diff
@@ -28,6 +28,8 @@ import (
 	"knative.dev/pkg/kmp"
 )
 
+const kafkaControllerSAName = "system:serviceaccount:knative-eventing:kafka-controller"
```

Comment:
Maybe this should be `system:serviceaccount:knative-eventing:eventing-controller`, or we should check for both?

From the failing tests:
```
Failed to sync physical Channel: admission webhook \"validation.webhook.kafka.eventing.knative.dev\" denied the request: validation failed: Channel.Spec.Subscribers changed by user system:serviceaccount:knative-eventing:eventing-controller which was not the system:serviceaccount:knative-eventing:kafka-controller service account
```